src/utils/money.cljs

```diff
@@ -204,10 +204,14 @@
       (with-precision 2)
       str))
 
-(defn add
+(defn- add*
   [bn1 n2]
   (.add ^js bn1 n2))
 
+(def add
+  "Add with defaults, this version is able to receive `nil` and takes them as 0."
+  (fnil add* (bignumber 0) (bignumber 0)))
+
```

Comment:
Added a more flexible version of `add`.
I'd really like to get your feedback about this change.

Comparison between this version and the previous one:
```clojure

(reduce money/add [])
;; Before:
;; Exception about null has no .add method
;; Now:
;; => (bignumber 0)

(reduce money/add [(money/bignumber 10)])
;; Before:
;; => (bignumber 10)
;; Now:
;; => (bignumber 10)

(reduce money/add [(money/bignumber 10) (money/bignumber 20) (money/bignumber 30)])
;; Before:
;; => (bignumber 60)
;; Now:
;; => (bignumber 60)

(money/add (money/bignumber 10))
;; Before:
;; Exception about no method plus() found in null
;; Now:
;; => (bignumber 10)

(money/add (money/bignumber 10) nil)
;; Before:
;; Exception about no method plus() found in null
;; Now:
;; => (bignumber 10)

(money/add nil (money/bignumber 10))
;; Before:
;; Exception about null has no .add method
;; Now:
;; => (bignumber 10)

(money/add nil nil)
;; Before:
;; Exception about null has no .add method
;; Now:
;; => (bignumber 0)
```

Reply:
This seems like a perfect solution to me for dealing with `nil` values as we should not allow the app to crash on `nil` values. 

Thanks for comparsion @ulisesmac!